Dockerfile.build

```diff
@@ -2,10 +2,18 @@ FROM golang:latest
 MAINTAINER Jess Frazelle <jess@docker.com>
 
 RUN go get github.com/Sirupsen/logrus && \
-    go get github.com/crosbymichael/octokat
+    go get github.com/crosbymichael/octokat && \
+	go get github.com/tobi/airbrake-go && \
+	go get github.com/bugsnag/bugsnag-go && \
+	go get github.com/getsentry/raven-go && \
+	go get gopkg.in/gemnasium/logrus-airbrake-hook.v2 && \
+	go get github.com/revel/revel && \
```

Comment:
means that I'm not constantly re-fetching the req's after the COPY. (my internet is unfun, so I use the build cache lots.